scripts/build_shapes.py

```diff
@@ -80,7 +81,64 @@ def download_GADM(country_code, update=False, out_logging=False):
     return GADM_inputfile_gpkg, GADM_filename
 
 
-def get_GADM_layer(country_list, layer_id, geo_crs, update=False, outlogging=False):
+def replace_nonstd_codes(row, col, country_code, keep_cond=True):
+    if row["GID_0"] != country_code:
+        return country_name_2_two_digits(row["COUNTRY"])
+    else:
+        return row["GID_0"]
+
+
+def filter_gadm(geodf, layer, cc, output_nonstd_to_csv=True, keep_all_areas=True):
+
+    # convert country name representation of the main country (GID_0 column)
+    geodf["GID_0"] = geodf["GID_0"].map(three_2_two_digits_country)
+
+    if keep_all_areas:
```

Comment:
Instead of keep_all_areas, as discussed below, we can pass directly the option fix_contented_areas and revise this if as:
```python
if fix_contented_areas == "drop":
...
else:
   if fix_contended_areas != "set_by_country":
      logger.warning("Option contented areas not implemented; fallback to set_by_country")  (message to be improved)
    ...
```

Reply:
Yeah, and that could would do that in a reversed order:
- if the user specifies the option drop, the drop code applies (the first ...)
- otherwise the option set_by_country applies.
   However, if the option specified by the user does not match "set_by_country" a message must be thrown to notify that the desired option does not exist.
   An error may be thrown instead

Reply:
Done